### DISCUSSION TITLE
Dedicated shortcut for "Did you mean" result

### DISCUSSION THREAD

**Discussion by moritzgvt** [infokiller/web-search-navigator#333] [General]
Currently there is no option to select the "Did you mean" result Google provides, when there's a typo in the searched phrase.

Workaround:
A current workaround to this is to hit "Shift+Tab"on the first search result. This will focus on the "Did you mean" result. Now press enter for submission. However that won't work when a sponsored result is shown before the proper results (which is the case disturbingly often).

Solution:
As  a solution I would suggest to have the "Did you mean" result selected by a dedicated shortcut. That would enable the user to correct the entered search query really fast.

**Comment by infokiller** [comment]
If the "Did you mean" link will be added to the navigatable search results (so that you can press down/j to reach it like a regular link), would it solve your issue?

**Comment by moritzgvt** [comment]
That'd be sufficient I think. But I wouldn't change it to the first focussed element – as it's really handy to just hit enter to select the first result.

**Comment by infokiller** [comment]
I've been thinking about this and I'm not sure what's the best UI for this. As you say, changing it to the first focused element may be annoying for some users. I could add an option to control this (whether it should be the first focused element or not), would that work for you?
A dedicated shortcut is possible (as in your original suggestion) but I would guess not many users will use it this way.

BTW, how often do you get "Did you mean"? in my usage, I don't get it much because even if I have typos in my query, I usually get it automatically, i.e. "Showing results for X".

Anyway, unless this gets more upvotes, it will probably take a while for me to implement it, but PRs are welcome.

**Comment by moritzgvt** [comment]
I think the best outcome for me as a user would be a dedicated shortcut.

Personally I get the "Did you mean" option quite often … don't know if this is dependent on the locale maybe or if I just need to learn typing without making mistakes 😅  

Let's see, if some other users also have this problem or if it's just me. Won't make sense to have that changed for one user only. Maybe I find some time to create a PR In the meanwhile. 

Thanks for looking into it! I appreciate.
